Products/CMFPlone/browser/templates/plone-overview.pt

```diff
@@ -31,14 +31,9 @@
                 i18n:attributes="alt" /></p>
         <h1 i18n:translate="">Plone is up and running.</h1>
         <p class="lead">
-            <span i18n:translate="label_plone_com_description"> For an introduction to Plone, success stories, demos, providers, visit</span>
-            <a href="http://plone.com"
-              title="Plone.com"
-              target="_blank"
-              i18n:attributes="title label_plone_com_title;">plone.com</a>.
-            <span i18n:translate="label_plone_org_description"> For documentation, add-ons, support, community, visit</span>
+            <span i18n:translate="label_plone_org_description"> For an introduction to Plone, documentation, demos, add-ons, support, and community, visit</span>
             <a href="http://plone.org"
-              title="Plone Community Home"
+              title="Plone Homepage"
```

Comment:
I pulled this from the `og:title` meta tag.

```suggestion
              title="Plone CMS: Open Source Content Management"
```

Reply:
@skleinfeldt @rioksane do we want to emphasize the Plone _Community_ aspect here? Or like @stevepiercy suggests: synchronize the title tag for plone.org everywhere? Either works for me.

Reply:
Also see here: https://github.com/collective/plone.app.locales/blob/master/plone/app/locales/locales/plone.pot#L12297

Reply:
Does this need to be removed?

https://github.com/collective/plone.app.locales/blob/master/plone/app/locales/locales/plone.pot#L12287-L12290

Reply:
@stevepiercy I do think... As it shows the same `message-id` as changed `title` in (Products/CMFPlone/browser/templates/plone-overview.pt)

Reply:
Is this possible instead?

```
#. Default: "Plone.org"
#: CMFPlone/browser/templates/plone-overview.pt:35
#| msgid "label_plone_com_title"
msgid "label_plone_org_title"
msgstr ""
```

Reply:
@mtrebron better!

Reply:
Sorry for the scattered thoughts: if it is only used in the plone-overview template, maybe better to remove altogether.

Reply:
We probably need some expert opinion from the translators team. @erral l ? I am not sure if plone.pot is generated in some way.

Reply:
What to remove? Sir...

Reply:
I would suggest reverting this back to "Plone Community Home" and create a new separate issue in collective/plone.app.locales to deal with the .pot changes

Reply:
Okay... I'll revert them soon!